main.js

```diff
@@ -636,25 +636,26 @@
 
             results.forEach(function (result) {
                 xml += indent.repeat(2) + "<testcase classname='" + classname + "'";
-                xml += " name='" + result.name + "' time='" + result.time + "'>\n";
+                if ( typeof(result) === "object" ) {
+                    xml += " name='" + result.name + "' time='" + result.time + "'";
```

Comment:
This could "look cooler" with the new ES6 template literals, but obviously this is not important right now
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals